bench/deno.js

```diff
@@ -1,17 +1,52 @@
-import { createCanvas } from "../mod.ts";
-// import { createCanvas as createCanvasWasm } from "https://deno.land/x/canvas@v1.4.1/mod.ts";
-import { draw } from "./draw.mjs";
+import { createCanvas, Image } from "../mod.ts";
+import {
+  createCanvas as createCanvasWasm,
+  loadImage,
+} from "https://deno.land/x/canvas@v1.4.1/mod.ts";
+import { draw, drawGradient, drawImage } from "./draw.mjs";
 
-Deno.bench("deno skia_canvas", () => {
+Deno.bench("encoding: deno skia_canvas", () => {
   const canvas = createCanvas(1024, 768);
   const ctx = canvas.getContext("2d");
   draw(ctx);
   canvas.encode("png");
 });
 
-// Deno.bench("deno canvaskit-wasm", () => {
-//   const canvas = createCanvasWasm(1024, 768);
-//   const ctx = canvas.getContext("2d");
-//   draw(ctx);
-//   canvas.toBuffer("image/png");
-// });
+Deno.bench("encoding: deno canvaskit-wasm", () => {
+  const canvas = createCanvasWasm(1024, 768);
+  const ctx = canvas.getContext("2d");
+  draw(ctx);
+  canvas.toBuffer("image/png");
+});
+
+Deno.bench("gradient: deno skia_canvas", () => {
+  const canvas = createCanvas(1024, 512);
+  const ctx = canvas.getContext("2d");
+  drawGradient(ctx);
+  canvas.encode("png");
+});
+
+Deno.bench("gradient: deno canvaskit-wasm", () => {
+  const canvas = createCanvasWasm(1024, 512);
+  const ctx = canvas.getContext("2d");
+  drawGradient(ctx);
+  canvas.toBuffer("image/png");
+});
+
+Deno.bench("images: deno skia_canvas", async () => {
+  const canvas = createCanvas(1000, 1400);
+  const ctx = canvas.getContext("2d");
+  const skiaImage = await Image.load("./testdata/skia_logo.png");
```

Comment:
use `new Image(path)` here, it'll skip deno layer entirely and load image natively

Reply:
Sounds good, let me update the benchmark.

Reply:
Did not look like it had a huge effect, the results are ~5 ms of before.